io.dbeaver.DBeaverCommunity.Client.mariadb.json

```diff
@@ -0,0 +1,38 @@
+{
+  "app-id": "io.dbeaver.DBeaverCommunity.Client.mariadb",
+  "runtime": "io.dbeaver.DBeaverCommunity",
+  "sdk": "org.freedesktop.Sdk//18.08",
```

Comment:
could you show this as a separate runtime-version please?

Reply:
Sorry, I can't figure out what do you want :)
add `runtime-version` tag for SDK?

Reply:
These really were not ready for pull yet. That's why I didn't make the pulls myself :)

https://github.com/flathub/io.dbeaver.DBeaverCommunity/pull/23#issuecomment-500238332

I'll have a look at them again in the morning.

Reply:
@krashekspres Please see krashekspres/flathub#1